dali/pipeline/operators/decoder/nvjpeg_decoder_decoupled_api.h

```diff
@@ -208,6 +203,14 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
           info.widths[0] = crop_window.w;
           info.heights[0] = crop_window.h;
         }
+
+        if (ShouldUseHybridHuffman(info, input_data, in_size, hybrid_huffman_threshold_)) {
```

Comment:
wondering: any reason for moving this code?

Reply:
ShouldUseHybridHuffman uses decoded image dimensions to decide if it is above the threshold set. It takes crop window into account, but before that change crop window was set to 0 - not used.

Reply:
Done